refmt_test/backportSyntax.t/input.re

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
I'm looking for a better name for this test

Reply:
backport-let-syntax?